src/Integration/Binding/CSharpVBProjectBinder.cs

```diff
@@ -87,19 +97,27 @@ public bool IsBindingRequired(BindingConfiguration binding, Project project)
 
         private bool IsFullyBoundProject(BindingConfiguration binding, Project project, Language language)
         {
-            if (!IsSolutionBound(binding, language, out var solutionRuleSetFilePath, out var additionalFilePath))
```

Comment:
The behaviour should be exactly the same as before. However, I had to refactor slightly to capture the value(s) to log. The checks are now all in the form:

```
var someCondition = CheckCondition();
logger.LogDebug( log the result);

if (!someCondition)
{
  return;
}

```